webpack/assets/javascripts/react_app/components/DiffView/DiffContainer.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ButtonGroup, Button } from 'patternfly-react';
+
+import DiffView from './DiffView';
+import './diffview.scss';
+
+class DiffContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      viewType: 'split',
+    };
+  }
+
+  onUnifiedClick() {
+    this.setState({ viewType: 'unified' });
+  }
+
+  onSplitClick() {
+    this.setState({ viewType: 'split' });
+  }
+
+  render() {
+    const { oldText, newText } = this.props;
+    return (
+      <div id="diff-container">
+        <div id="diff-radio-buttons">
+          <ButtonGroup>
+            <Button
+              className={this.state.viewType === 'split' ? 'diff-button active' : 'diff-button'}
```

Comment:
you can use `classnames` for a conditional class
https://github.com/JedWatson/classnames